src/sdk/base/mediastream-factory.js

```diff
@@ -20,7 +20,7 @@ export class AudioTrackConstraints {
   // eslint-disable-next-line require-jsdoc
   constructor(source) {
     if (!Object.values(MediaFormatModule.AudioSourceInfo)
-        .some((v) => v === source)) {
+      .some((v) => v === source)) {
```

Comment:
Please do not making unnecessary changes.

Reply:
accepted, revert the format

Reply:
Please remove other unnecessary changes as well.